### ISSUE TITLE
Feature: Add support for custom GraphQL scalars `@gql(scalars: { website: "AWSURL" })`

### ISSUE THREAD

**Issue by Tenrys** [maoosi/prisma-appsync#88]
Right now it seems like only fields that are named `url` or `email` have been hardcoded to use the `AWSURL` and `AWSEmail` scalar types from AWS, however I would like to be able to use these types on fields like `urlPhoto` or something like `secondaryEmail`.

As seen here: 
https://github.com/maoosi/prisma-appsync/blob/37b4a68d76c891c8ea0a818c358691629ba1c5bc/packages/generator/src/compiler.ts#L859-L868

Would it be possible to use triple slash comments in the Prisma schema to define which fields turn up as what scalar types in the GraphQL schema?

**Comment by maoosi** [comment]
Possibly yes, but would need to find the right syntax...

There is already a custom notation in place (not documented yet) that allows customising the generated GraphQL schema.

```prisma
/// @gql(queries: { list: null, count: "countyPosts" }, fields: { text: null }, subscriptions: null)
model Post {
    text  String?
    urlPhoto  String?
}
```

The above notation equals to:
- Remove `listPosts`
- Rename `countPosts` to `countyPosts`
- Remove `text` field
- Remove all subscriptions

One option could be:

```prisma
/// @gql(scalars: { urlPhoto: "AWSURL" })
model Post {
    urlPhoto  String?
}
```

Implementing the above would be quite simple, as the code to read the `@gql` directive is already in place.

What do you think?

**Comment by Tenrys** [comment]
I think that makes sense yes! Until documentation gets written, is there a way to know where exactly the code involving that custom notation is in place?

**Comment by maoosi** [comment]
Parsing the `@gql` directive as a JSON object is done here:

https://github.com/maoosi/prisma-appsync/blob/37b4a68d76c891c8ea0a818c358691629ba1c5bc/packages/generator/src/compiler.ts#L367-L375

The following is the config object where to add `_scalars: gqlObject?.scalars || {}`:

https://github.com/maoosi/prisma-appsync/blob/37b4a68d76c891c8ea0a818c358691629ba1c5bc/packages/generator/src/compiler.ts#L393-L406

Here is where the check to remove fields is done:

https://github.com/maoosi/prisma-appsync/blob/37b4a68d76c891c8ea0a818c358691629ba1c5bc/packages/generator/src/compiler.ts#L544-L567

For example, we could do:

```tsx
const scalar = gqlConfig?._scalars?.[field.name] || this.getFieldScalar(field)
```

**Comment by maoosi** [comment]
@Tenrys do you want to work on a PR for this feature?

**Comment by maoosi** [comment]
Will be released as part of `1.0.0-rc.6` (you can also try it now with `prisma-appsync@1.0.0-preview.6.6`).
